rules/filter.smk

```diff
@@ -21,113 +20,126 @@ rule select_bad_samples:
     input:
         psc=rules.vcf_stats.output.psc
     output:
-        bad_samples="vcf/lifted_vcf.badsamples",
-        report="results/bad_samples_report.tsv"
-    log: "logs/vcf/select_bad_samples.log"
+        bad_samples="vcf/{segment}_lifted_vcf.badsamples",
+        report="results/{segment}_bad_samples_report.tsv"
+    log: "logs/vcf/{segment}_select_bad_samples.log"
     params:
-        samples="vcf/merged_lifted.vcf.samples"
+        samples="vcf/{segment}_merged_lifted.vcf.samples"
     conda:
         "../envs/evaluation.yaml"
     script:
         "../scripts/select_bad_samples.py"
 
 rule plink_filter:
     input:
-        vcf="vcf/merged_lifted_id.vcf.gz",
+        vcf="vcf/{segment}_merged_lifted_id.vcf.gz",
         bad_samples=rules.select_bad_samples.output.bad_samples
-    output: expand("plink/merged_filter.{ext}", ext=PLINK_FORMATS)
+    output:
+        bed = temp("plink/{segment}_merged_filter.bed"),
+        bim = temp("plink/{segment}_merged_filter.bim"),
+        fam = temp("plink/{segment}_merged_filter.fam")
     conda:
         "../envs/plink.yaml"
     params:
-        input   = "merged",
-        out     = "merged_filter"
+        input   = "{segment}_merged",
```

Comment:
We should use just one type of quotes, `"` or `'`. I think we should prefer `'` for both python and snakemake code.

Reply:
Majority of code is written using `"`, i think we should stick with it, unless you got some reasons for such a change. Also i changed all `'` to `"`, thank you.